sql/session.go

```diff
@@ -120,6 +120,8 @@ type Session interface {
 	// SetViewRegistry sets the view registry for this session. Integrators should set a view registry if their database
 	// doesn't implement ViewDatabase and they want views created to persist across sessions.
 	SetViewRegistry(*ViewRegistry)
+	// WithConnectionId attaches the handler's connection ID to a new sql.Session separate from the session builder
```

Comment:
Comment does not make sense

Reply:
All the other Setter methods just modify in place instead of returning a copy. Should follow that pattern here unless there's a good reason not to.